Source/Core/VideoCommon/FramebufferShaderGen.cpp

```diff
@@ -570,6 +570,14 @@ std::string GenerateTextureReinterpretShader(TextureFormat from_format, TextureF
   }
   break;
 
+  case TextureFormat::I4:
+  {
+    ss << "  float4 temp_value = ";
+    EmitTextureLoad(ss, 0, "coords");
+    ss << ";\n"
+          "  raw_value = uint(temp_value.r * 15.0);\n";
+  }
```

Comment:
This seems to be missing a break.

Reply:
you are right. Can you let me know how to test this code path? this switch missing many formats. and there is also no code to dealing with same src and dest format.

Reply:
There's only a handful of games which rely on this "reinterpreting-texture-data-from-one-format-to-another" behavior. The expression you've added seems fine, but difficult to know if anything will actually use it.

As for the existing formats, fifoci should catch any regressions there.